docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CustomMappingSample.cs

```diff
@@ -10,71 +13,88 @@ public static void Example()
             var mlContext = new MLContext();
 
             // Get a small dataset as an IEnumerable and convert it to an IDataView.
-            var data = SamplesUtils.DatasetUtils.GetInfertData();
-            var trainData = mlContext.Data.LoadFromEnumerable(data);
+            var rawData = GetData();
```

Comment:
>GetData [](start = 26, length = 7)

if the dataset is has <10 rows, please define it inline instead of a method.

we only use a method for generating 100+ random data point (usually for trainer.)